tools/verdict-census.py

```
@@ -513,6 +539,39 @@ def self_test():
+        refuses = _fixture(d, "st_refuses.py",
+                           "import sys\n"
+                           "if '--self-test' in sys.argv:\n"
+                           "    print('VOID - no in-process control; run its paired suite',"
+                           " file=sys.stderr)\n"
+                           "    raise SystemExit(2)\n"
+                           "if len(sys.argv) > 1: raise SystemExit(2)\n"
+                           "raise SystemExit(0)")
+        refuses.write_text("#!/usr/bin/env python3\n"
+                           "# NO-SELF-TEST: controlled by its paired suite\n"
+                           + refuses.read_text().split("\n", 1)[1])
+        got, why = selftest_state(refuses, timeout=30)
```

Ruff (0.16.3)

[warning] Unpacked variable `why` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)